pkg/client/typed/dynamic/client.go

```diff
@@ -251,6 +245,18 @@ func (dynamicCodec) Encode(obj runtime.Object, w io.Writer) error {
 	return runtime.UnstructuredJSONScheme.Encode(obj, w)
 }
 
+// ContentConfig returns a restclient.ContentConfig for dynamic types.
+func ContentConfig() restclient.ContentConfig {
```

Comment:
This is just a move of what was already there. Would this be an easy fix, or something for another PR?